src/nominatim_db/clicmd/api.py

```diff
@@ -277,8 +279,7 @@ def run(self, args: NominatimArgs) -> int:
                                      layers=layers,
                                      address_details=True,  # needed for display name
                                      geometry_output=_get_geometry_output(args),
-                                     geometry_simplification=args.polygon_threshold,
-                                     locales=_get_locales(args, api.config.DEFAULT_LANGUAGE))
+                                     geometry_simplification=args.polygon_threshold)
```

Comment:
You have removed the locales parameter here and everywhere below but I don't see any code that uses your new localize() function instead.

Reply:
line 216 only applies to class APISearch. Here we are in class APIReverse. it needs its own separate call to localise(). And so does class APILookup and class APIDetails.